src/cmd/bootstrap.ts

```diff
@@ -0,0 +1,172 @@
+import { copyFileSync, existsSync, mkdirSync, readdirSync, rmSync, writeFileSync } from 'fs'
+import { copy } from 'fs-extra'
+import { copyFile } from 'fs/promises'
+import { Argv } from 'yargs'
+import { $ } from 'zx'
+import { OtomiDebugger, terminal } from '../common/debug'
+import { BasicArguments, ENV, loadYaml } from '../common/no-deps'
+import { cleanupHandler, otomi } from '../common/setup'
+import { ask } from '../common/zx-enhance'
+import { genSops } from './gen-sops'
+
+const fileName = 'bootstrap'
+let debug: OtomiDebugger
+const profileOptions = readdirSync('profiles').filter((val) => val !== 'common')
```

Comment:
Suppose I run this Github Action:
```
Run ENV_DIR=/home/app/stack/profiles/aws /usr/bin/node --experimental-specifier-resolution=node /home/app/stack/dist/otomi.js --<cmd>

```
This is the output:
```
node:fs:1380
  handleErrorFromBinding(ctx);
Error: ENOENT: no such file or directory, scandir 'profiles'
    at readdirSync (node:fs:1380:3)
    at file:///home/app/stack/dist/cmd/bootstrap.js:12:24 {
  errno: -2,
  syscall: 'scandir',
  code: 'ENOENT',
  path: 'profiles'
}

Error: Process completed with exit code 1.
```

However, I can set `working-directory: /home/app/stack`, which will set the CWD to `/home/app/stack`, and it will resolve the `ENV_DIR` path correctly. 

I expected that if I run `otomi.js` from an arbitrary path it would not break on the `readdirSync` error, so I wonder what the actual output should be when you do so? Should it throw this `readdirSync` error?

Reply:
Profiles no longer exists, so it isn't a problem anymore.
And yes, you can run it from an arbitrary path.
But when you run it from any folder that is `/**/otomi-core/**` it will go into development mode, and becomes a little more finicky.